examples/apis.json

```diff
@@ -53,7 +53,7 @@
     "args": ["context.Background()", "map[string]interface{}{}"]
   },
   "power_sensor": {
-    "func": "Voltage",
+    "func": "Power",
```

Comment:
@oliviamiller should we change this for all sdks?

Reply:
yeah probably, I can change the others.

Reply:
I don't think it *needs* to be changed for the other SDKs because I believe they're ok with assigning tuples to a single object but I'm not sure about TS. Still would be good for consistency